### DISCUSSION TITLE
Is there a way to pre-define the Window of the Waydroid.

### DISCUSSION THREAD

**Discussion by zak-admin** [waydroid/waydroid#1627] [Q&A] [answered]
I was able to pre-define the size of the window using the following commands.

```bash
waydroid prop set persist.waydroid.width 350 
waydroid prop set persist.waydroid.height 700
waydroid session stop
```

I would also like to have it given a definite space of my screen, and it is the right side of the screen. And I want my waydroid always to be active, in the right side of the screen, and my desktop apps will be functioning on the remaining side, as in the picture below.

![image](https://github.com/user-attachments/assets/76a094b5-01f8-414c-a0b4-28f78d763c10)

**Comment by zak-admin** [comment]
This Question is also related, https://github.com/waydroid/waydroid/discussions/1626

**Comment by aleasto** [comment, marked as answer]
It is not possible for an application to choose its own position on the screen.
This is the case for any application, not just Waydroid.
